skdate/src/date.sk

```diff
@@ -37,7 +37,7 @@ const daysInNonLeapYearMonth: Array<Int> = Array[
 class WeekDate(year: Int, week: Int, day: Day) uses Orderable {}
 
 base class Entity {
-  fun toCalendar(): Calendar;
+  fun toCalendar(tz: TZKind = TZLocal()): Calendar;
```

Comment:
Should the defaults not be utc for these params?

Reply:
It's the default of skdate library, date libraries used to be in local by default.